GameOfLife/Model/Game.cs

```diff
@@ -1,53 +1,71 @@
 ﻿namespace GameOfLife
 {
+    [Serializable]
     /// <summary>
     /// Mechanics of the game.
     /// </summary>
-    public class Game
+    public class Game : IGame
     {
         /// <summary>
         /// Array that will contain the game field.
         /// </summary>
-        public int [,] gameField;
+        public int[,] GameField { get; set; }
+        /// <summary>
+        /// Count of current game iterations.
+        /// </summary>
+        public int CountOfIteration { get; set; }
+        /// <summary>
+        /// Count of live cells in current generation.
+        /// </summary>
+        public int CountOfLiveCells { get; set; }
+
         /// <summary>
         /// Starts the game with first random generation.
         /// </summary>
         /// <param name="column">The number of columns entered by the user.</param>
         /// <param name="row">The number of rows entered by the user.</param>
-        public Game (int column, int row)
+        public Game(int column, int row)
         {
-            gameField = new int [column , row];
+            CountOfIteration = 0;
+            CountOfLiveCells = 0;
+            GameField = new int[column, row];
             SetRandomFirstGeneration();
         }
+
         /// <summary>
         /// Sets first random generation of live and dead cells in game field.
         /// </summary>
         public void SetRandomFirstGeneration()
         {
-            Random random = new Random();
-            for (int colomn = 0; colomn < gameField.GetLength(0); colomn++)
+            Random random = new();
+            for (int colomn = 0; colomn < GameField.GetLength(0); colomn++)
             {
-                for (int row = 0; row < gameField.GetLength(1); row++)
+                for (int row = 0; row < GameField.GetLength(1); row++)
                 {
-                    gameField[colomn , row] = random.Next(3) == 0 ? 1 : 0;
+                    GameField[colomn, row] = random.Next(3) == 0 ? 1 : 0;
                 }
             }
         }
+
         /// <summary>
         /// Gets the next generation of live and dead cells.
         /// </summary>
         public void GetNextGeneration()
         {
-            int[,] newGameField = new int[gameField.GetLength(0) , gameField.GetLength(1)];
-            for (int column = 0; column < gameField.GetLength(0); column++)
+            int[,] newGameField = new int[GameField.GetLength(0), GameField.GetLength(1)];
+            CountOfLiveCells = 0;
+            for (int column = 0; column < GameField.GetLength(0); column++)
             {
-                for (int row = 0; row < gameField.GetLength(1); row++)
+                for (int row = 0; row < GameField.GetLength(1); row++)
                 {
                     newGameField[column, row] = AliveCells(column, row);
```

Comment:
AliveCells is misleading name.

May be better WillCellBeAlive
or similar

Reply:
Method is renamed.